src/renderer/components/mulmo_viewer/tabs/slide_tab.vue

```
@@ -157,4 +158,44 @@
+      downloadImageFile(props.projectId, index, mulmoEvent.id);
+    }
+    if (mulmoEvent?.kind === "beat") {
+      if (mulmoEvent.sessionType === "audio") {
+        const index = props.project?.script?.beats?.findIndex((beat) => beat.id === mulmoEvent.id);
+        if (index === -1 || index === undefined) {
+          return;
+        }
+        downloadAudioFile(index, mulmoEvent.id);
+      }
+      if (mulmoEvent.sessionType === "multiLingual") {
+        updateMultiLingual();
```

ESLint

[error] 'updateMultiLingual' is not defined.

(no-undef)